### PR TITLE
feat: Adds Efficientnet B7 in the classification models

### PR DESCRIPTION
**Pull request by hasibzunair**
PR adds EfficientNet B7 model. 

Tried TF's EfficientNets (both` tf.keras.applications` and `tensorflow.keras`) but it does to seem to work for TPU's. I got a test ACC of 5.4% on the yogapose dataset.

### CODE REVIEW

File: decavision/model_training/tfrecords_image_classifier.py [hunk 1]
```diff
@@ -197,6 +197,11 @@ def _create_model(self, activation, hidden_size, dropout, l2_lambda):
             base_model = EfficientNetB5(weights='imagenet', include_top=False,
                                         input_shape=(*self.target_size, 3))
             base_model_last_block = 417  # last block 559, two blocks 417
+            
+        elif self.transfer_model == 'B7':
+            base_model = EfficientNetB7(weights='imagenet', include_top=False,
+                                        input_shape=(*self.target_size, 3))
+            base_model_last_block = None  # all layers trainable
```
Comment (yangobeil) [inline decavision/model_training/tfrecords_image_classifier.py:204]: Why are you not specifying the last two blocks?

**Comment by yangobeil** [comment]
You should modify the documentation as well to show that we can now use B7. From what I can see the pretrained models are listed in the train_example.rst file and in the docstring of the main class for classification.

Also I know it doesn't concern your modifications but can you take a look at the message from the bot and make the corresponding modifications?

Finally it looks like you have to add a semantic statement at the beginning of the PR title (like 'feat:') to satisfy a check before merging. Your options are listed at https://github.com/commitizen/conventional-commit-types/blob/master/index.json.

File: decavision/model_training/tfrecords_image_classifier.py (hunk 1 above)
Comment (hasibzunair) [reply decavision/model_training/tfrecords_image_classifier.py:204]: Whenever I used B7, I trained all layers as it gave better results, so I set it to `None`.

Comment (yangobeil) [reply decavision/model_training/tfrecords_image_classifier.py:204]: Interesting! How did the speed compare with just unfreezing the last blocks?

Comment (hasibzunair) [reply decavision/model_training/tfrecords_image_classifier.py:204]: It was marginal from my initial experiments, both took some time to train!

**Review by yangobeil — APPROVED** [review]
I tested it quickly and it seems to work fine.

**Comment by hasibzunair** [comment]
Ready for merge!?

**Comment by yangobeil** [comment]
> Ready for merge!?

Yes I accepted the PR so the next step is to merge, which you should be allowed to do.

**Pull request merged**